homeworks/dima.zapisochny_dima-88/homework_5/test3.js

```diff
@@ -0,0 +1,12 @@
+const objToCopy = { b: 'c', d: { e: 'f' } };
+
+function innerCopy(obj) {
+    const objCopy = {};
+    Object.keys(obj).forEach((key) => {
+        objCopy[key] = obj[key];
+    });
```

Comment:
this will not do deep copy of objects... `objToCopy.d` and `objCopy.d` will reference the same object